packages/coordinator/ts/sessionKeys/dto.ts

```diff
@@ -0,0 +1,33 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { Transform } from "class-transformer";
+import { IsEnum, IsEthereumAddress } from "class-validator";
+
+import type { Hex } from "viem";
+
+import { ESupportedNetworks, transformToString } from "../common";
+
+/**
+ * Data transfer object for Deactivate session key
+ */
+export class DeactivateSessionKeyDto {
+  /**
+   * Session key address
+   */
+  @ApiProperty({
+    description: "Session key address",
+    type: String,
+  })
+  @IsEthereumAddress()
+  sessionKeyAddress!: Hex;
+
+  /**
+   * Chain Name
+   */
+  @ApiProperty({
+    description: "Chain to which to deploy the contract(s)",
```

Comment:
guess this comment is not accurate? it's DeactivateSessionKeyDto so maybe should be related to deactivation?